tests/unit/app/test_lamba_handler.py

```diff
@@ -0,0 +1,34 @@
+import unittest
```

Comment:
typo:  Name should be "test_lambda_handler.py" not "test_lamba_handler.py"

Reply:
fixed